examples/common.cpp

```diff
@@ -256,3 +265,47 @@ std::vector<llama_token> llama_tokenize(struct llama_context * ctx, const std::s
 
     return res;
 }
+
+/* Keep track of current color of output, and emit ANSI code if it changes. */
+void set_console_color(console_state* con_st, console_color_t color) {
```

Comment:
Consider replacing `con_st` with a reference to make it clear that NULL is not an acceptable value